app/models/corporate_information_page.rb

```diff
@@ -175,13 +175,15 @@ def alternative_format_provider_required?
   end
 
   def base_path
-    if worldwide_organisation.present?
-      url = edition_worldwide_organisation.worldwide_organisation.base_path.to_s + "/about/#{slug}"
-      url.gsub("/about/about", "")
-    elsif organisation.present?
-      url = edition_organisation.organisation.base_path.to_s + "/about/#{slug}"
-      url.gsub("/about/about", "/about")
-    end
+    return if owning_organisation.blank?
+
+    url = owning_organisation.base_path + "/about/#{slug}"
+    # The corporate information pages associated with worldwide organisations are not rendered
```

Comment:
We tend to try and avoid comments if possible. I'd suggest adding a line or two to the commit message so that we can remove these.

Reply:
I thought this might be warranted because it's quite confusing. Happy to remove it if you disagree!